src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1075,10 +1075,12 @@ private static void validatePropertySectionPropertyTokens(String fileName, Strin
         final String defaultTokenText = columns.get(3).getTextContent().trim();
         final String expectedDefaultTokenText = CheckUtil.getTokenText(check.getDefaultTokens(),
                 check.getRequiredTokens());
+        // Until https://github.com/checkstyle/checkstyle/issues/13876
         if (expectedDefaultTokenText.isEmpty()) {
-            assertWithMessage("Empty tokens should have 'empty' string in xdoc")
-                .that(defaultTokenText)
-                .isEqualTo("empty");
+            assertWithMessage("Empty tokens should have"
+                    + " '.', '<code>&quot;&quot;</code>','empty' string in javadoc")
+                    .that(".".equals(defaultTokenText) || "empty".equals(defaultTokenText))
```

Comment:
`<code>&quot;&quot;</code>','empty' string in xdoc`

Because you use xml tags is begging if message

Reply:
@romani , the requested changes are bit unclear. Can you help me on this?

Reply:
```
"Empty tokens should have"
                    + " '.', '<code>&quot;&quot;</code>' string in javadoc"
```
Is this the requested change?

Reply:
Make code as:
```
assertWithMessage("Empty tokens should have"
                    + " '.', '<code>&quot;&quot;</code>','empty' string in xdoc")
                    .that
```